src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -49,7 +49,7 @@ public void setUp() {
     @Override
     protected String getPath(String filename) throws IOException {
         return super.getPath("checks" + File.separator
-                + "whitespace" + File.separator + filename);
```

Comment:
@rnveach 
Oh, yes. You are right. My fault. SeparatorWrap is a part of the checks which are in whitespace group.
http://checkstyle.sourceforge.net/config_whitespace.html